### DISCUSSION TITLE
How to file a support ticket on Partner Center?

### DISCUSSION THREAD

**Discussion by sonakshimore** [microsoft/MicrosoftEdge-Extensions#53] [Q&A] [answered]
I am unable to verify my developer account and want to contact the support team for assistance on this. How do I proceed?

**Comment by sonakshimore** [comment, marked as answer]
You can create a support ticket directly via your Partner Center account. Follow the below steps to proceed further:

1. On your Partner Center dashboard, at the bottom of the page, click on the **Get support** link as indicated in the below snippet
<img width="852" alt="step 2" src="https://user-images.githubusercontent.com/85096285/191159504-17af6c0c-7154-4ea5-8888-8057e8d467a4.png">

2. You will be taken to a new window which opens the **New support request** form
<img width="329" alt="image" src="https://user-images.githubusercontent.com/85096285/191159260-ff63febf-a895-4082-9d20-3057972abd3f.png">

3. Fill out the necessary details of the issue you are facing:

- **Issue Title** - Add a short summary of your issue
- **Your First Name**
- **Your Last Name**
- **Email Address** - Enter your registered email address
- **Preferred contact email address** - Enter the email address you would like to be contacted on by the support team
- **Time Zone** - Select your time zone
- **A detailed description of your issue or question along with any error messages** - Provide a detailed explanation of the question or issue you need assistance with. If you receive any error message, do ensure to include it here. Do not include any additional personal information when submitting your comments.

4. Once you have entered all the necessary information in all the fields, click on **Submit**
5. Your support ticket is now raised successfully

**Comment by mpulukkinen** [comment]
I'm having similar issues, with the new "AI assistant" and tons of different categories, I just can't find the proper place to tell them "your trial mode thing seems to give users full access" :D Whenever I find remotely relative category, i just seem to get message "not our problem, report somewhere else". There's like total lack of support for the actual apps & games

**Reply by kirkchenfinquery** [reply]
I have the same issue as you. My help is in a section shared with AI assistant, and it requires a Workspace and Problem Type field. None of the problem type seems applicable. And when I did submit a ticket to the one I can find, they just said it's not their department.

I am not sure where to get help at all.

**Reply by mpulukkinen** [reply]
This seems to plaque the whole Microsoft: I got my advertisement account banned while ago, the gave me a huge list of possible issues but no single action what I should fix. After couple of ping pongs support lifted the ban just for the account to get banned again next day. One more email to them, next message is "sorry, can't do a thing, not telling more detais" and total radio silence after that. Send multiple emails and no answer. They stole 20e from me, I added some cash for the apps and never got to use them XD

**Reply by ManikanthMSFT** [reply]
Hi @mpulukkinen, sorry for the inconvenience caused. We request you to please send us an [email ](ext_dev_support@microsoft.com)so that we will review your concern and will assist you deliberately.

**Comment by venkatasai5171** [comment]
Hello,

I’m Venkat from the Microsoft Edge Extension Developer Support Team.

I hope your query has been resolved. As there has been no further response, I’m closing this discussion.

If you need to raise a support request on Partner Center, you can follow the steps mentioned above. Alternatively, you may:

- Raise a bug on GitHub, or

- Send a support request via email to ext_dev_support@microsoft.com

Regards,
Venkat

**Comment by blaiz2023** [comment]
I have a Partner Center account and trying to get *** ANY SANE HELP *** from Microsoft is like trying to get blood out of stone.  I've been at it for nearly a month now.  Trying to get help with an app that's supposedly failed MS Store certification, but tech support won't talk to me, and trying to file another support ticket is insane crazy hard - wish there was an option to nuke the smart ass "AI Assitant" which is just another way of saying go away we don;'t want to help you. Seriously wondering if it's worth it submitting any more apps to the app store.  The stone cold wall of silence from Tech Support is anything but encouraging.

**Reply by mpulukkinen** [reply]
Try here: https://discord.gg/76fkpfk9C Looks like all companis have silently moved their support to discord and forgot to inform everyone 🤣

**Reply by blaiz2023** [reply]
Thanks for your help.  Will check it out.
